app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<form id="login_form" class="form-horizontal" action="/sessions" method="post">
+  <input name="authenticity_token" value="<%= form_authenticity_token %>" type="hidden">
+  <div class="form-group">
+    <div class="col-sm-10">
+      <label for="username" class="col-sm-2 control-label">Username:</label>
+      <input class="form-control" type="text" name="session[username]" value="">
+    </div>
+  </div>
+  <div class="form-group">
+      <div class="col-sm-10">
+        <label for="password" class="col-sm-2 control-label">Password:</label>
+        <input class="form-control" type="password" name="session[password]" value="">
+      </div>
+  </div>
+  <div class="error_div">
+    <ul class="error_login">
+      <% if @errors %>
+        <% @errors.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      <% end %>
+    </ul>
+  </div>
+  <br/>
```

Comment:
think this should just be `<br>`
or use CSS to increase margins for best practice. From MDN: "The HTML element line break <br> produces a line break in text (carriage-return). It is useful for writing a poem or an address, where the division of lines is significant.

Do not use <br> to increase the gap between lines of text; use the CSS margin property or the <p> element."